docroot/themes/custom/dcp/sass/components/_header.scss

```diff
@@ -0,0 +1,268 @@
+.region-header {
+  display: flex;
+  background: white;
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);
+  min-height: 97px;
+  padding: 18px 51px 16px 60px;
+  align-items: center;
+  justify-content: center;
+  .block-system-branding-block {
+    width: 118px;
+    height: 63px;
+    @include breakpoint(tablet) {
+      width: 88px;
+      height: 47px;
+    }
+  }
+  > nav[role="navigation"] {
+    h2 {
+      display: none;
+    }
+    &.menu--main {
+      a {
+        font-family: Circular Air;
+        font-size: 12px;
+        padding-bottom: 11px;
+        line-height: 15px;
+        letter-spacing: 1px;
+        text-transform: uppercase;
+        color: #313131;
+        text-decoration: none;
+        &.is-active {
+          color: #316CF4;
+        }
+      }
+      >.menu {
+        display: flex;
+        >.menu-item{
+          margin-right: 35px;
+          @include breakpoint(laptop) {
+            margin-right: 20px;
+          }
+        }
+      }
+      @include breakpoint(tablet) {
+        display: none;
+      }
+    }
+    &.menu--account {
+      min-width: 124px;
+      text-align: right;
+      ul > li {
+        width: 124px;
+      }
+      a{ font-family: Circular Air;
+        font-size: 12px;
+        line-height: 15px;
+        padding-bottom: 11px;
+        letter-spacing: 1px;
+        text-transform: uppercase;
+        text-decoration: none;
+        color: #313131;
+      }
+      @include breakpoint(tablet) {
+        display: none;
+      }
+    }
+    .menu-item--expanded {
+      position: relative;
+      display: inline-block;
+      .menu {
+        width: calc(100% - 10px);
+        right: 0;
+        display: none;
+        text-align: left;
+        cursor: pointer;
+        position: absolute;
+        z-index: 1;
+        background: white;
+        margin-top: 11px;
+        box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.3);
+        border-radius: 2px;
+      }
+      li {
+        >a {
+          padding: 11px 21px 11px 12px;
+          letter-spacing: 1px;
+          font-family: Circular Air;
+          font-size: 12px;
+          line-height: 10px;
+          text-decoration: none;
+          color: #313131;
+          display: block;
+        }
+      }
+    }
+    .menu-item--expanded:hover .menu {
+      display: block;
+    }
+  }
+  .block-system-branding-block {
+    margin: 0 auto 0 0;
+  }
+  img {
+    width: 100%;
+  }
+  #block-logintext {
+    padding-left: 50px;
+    a {
+      background: #FF2370;
+      border-radius: 23.5px;
+      font-family: Circular Air;
```

Comment:
@theyogendraDanwar - i think, you might have to change the font family as per the fonts added. Please refer this file -  https://github.com/nikitaaswani/dcp/blob/b310857390fc3386178e5a2a08d90299e808487b/docroot/themes/custom/dcp/sass/utilities/_typography.scss